server/akka-http-server/src/main/scala/sttp/tapir/server/akkahttp/AkkaHttpServerInterpreter.scala

```diff
@@ -44,6 +48,28 @@ trait AkkaHttpServerInterpreter {
   ): Route = {
     new EndpointToAkkaServer(serverOptions).toRoute(serverEndpoints)
   }
+
+  def serialiseSSEToBytes: Source[ServerSentEvent, Any] => AkkaStreams.BinaryStream = sseStream =>
+    sseStream.map(sse => {
+      ByteString(s"${composeSSE(sse)}\n\n")
+    })
+
+  private def composeSSE(sse: ServerSentEvent) = {
```

Comment:
maybe we could move this logic to sttp-model?

Reply:
Yeah, probably a better place. PR for sttp-model: https://github.com/softwaremill/sttp-model/pull/75/files

Reply:
sttp-model is bumped, composeSSE is deleted and toString is used instead